CNN/Scripts/ScriptsSetup.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+echo "Copying scripts to their respective directories"
+echo " "
+echo "Copying Autorun.sh, Randomizer.sh and Timediff.sh scripts to the CNN directory"
+cp Autorun.sh ../
+cp Timediff.sh ../
+cp Randomizer.sh ../Model/Data/Train/
```

Comment:
The script is still moving files, this is unrequired and can be changed to run from the Scripts directory., To confirm, we do not need to move any files.